aospy/plotting.py

```diff
@@ -346,22 +345,39 @@ def _set_axes_labels(self):
             self.ax.set_title(self.ax_title, fontsize='small')
         # Axis panel labels, i.e. (a), (b), (c), etc.
         if self.ax_label:
+            if self.ax_label == 'auto':
+                text = '({})'.format(tuple('abcdefghijklmnopqrs')[self.ax_num])
```

Comment:
Could this line be simplified to something like this?
```python
text = '({})'.format(string.ascii_letters[self.ax_num])
```

Albeit it would require an extra import statement for ```string```; regardless I don't see the need to cast the string as a tuple.  String indexing should be fine here.